ReportingServicesTools/Functions/Security/Rest/Get-RsRestItemAccessPolicy.ps1

```diff
@@ -81,6 +81,10 @@ function Get-RsRestItemAccessPolicy
     Begin
     {
         $WebSession = New-RsRestSessionHelper -BoundParameters $PSBoundParameters
+        if ($Credential.Username -ne $WebSession.Credentials.Username) {
```

Comment:
I have been thinking about this implementation , and one thing that seems a risk is that when explicitly is provided the $credential parameter it would expect it to be honored, however if there is a websession involved there is a chance that will be overwritten by the WebSession credentials

Consider instead of comparing if they are different , check if the $credential is not provided (will a nullcheck work?) and in that case can be overridden by the websession credentials

Reply:
I see what you mean, I thought specifying the Credential parameter as well as WebSession would update the credentials in the websession, but some debugging shows that these two values can be different within the call.

Just submitted a change using $null checks